packages/framework/src/Component/Collector/ShopsysFrameworkDataCollector.php

```diff
@@ -34,12 +35,17 @@ public function collect(Request $request, Response $response, ?Throwable $except
             'version' => ShopsysFrameworkBundle::VERSION,
             'docsVersion' => $this->resolveDocsVersion(ShopsysFrameworkBundle::VERSION),
             'domains' => $this->domain->getAll(),
-            'currentDomainId' => $this->domain->getId(),
-            'currentDomainName' => $this->domain->getName(),
-            'currentDomainLocale' => $this->domain->getLocale(),
             'systemTimeZone' => date_default_timezone_get(),
-            'displayTimeZone' => $this->displayTimeZoneProvider->getDisplayTimeZoneByDomainId($this->domain->getId())->getName(),
         ];
+
+        try {
+            $this->data['currentDomainId'] = $this->domain->getId();
+            $this->data['currentDomainName'] = $this->domain->getName();
+            $this->data['currentDomainLocale'] = $this->domain->getLocale();
+            $this->data['displayTimeZone'] = $this->displayTimeZoneProvider->getDisplayTimeZoneByDomainId($this->domain->getId())->getName();
+        } catch (NoDomainSelectedException) {
+            return;
```

Comment:
How does it behave when no domain is selected? 
getter methods are strictly typed, so my guess is it will break just a moment later. 

What about setting some default values ('-' for strings, 0 for int maybe)